boilerplate/_lib/ensure.sh

```diff
@@ -22,6 +36,54 @@ golangci-lint)
         curl -sfL "${DOWNLOAD_URL}" | tar -C "${GOPATH}/bin" -zx --strip-components=1 "golangci-lint-${GOLANGCI_LINT_VERSION}-${GOOS}-amd64/golangci-lint"
     fi
     ;;
+
+operator-sdk)
+    #########################################################
+    # Ensure operator-sdk is installed at the desired version
+    # When done, ./.operator-sdk/bin/operator-sdk will be a
+    # symlink to the appropriate executable.
+    #########################################################
+    # First discover the desired version from go.mod
+    # The following properly takes `replace` directives into account.
+    wantver=$(go list -json -m github.com/operator-framework/operator-sdk | jq -r 'if .Replace != null then .Replace.Version else .Version end')
```

Comment:
A couple of things, this will list a version for the operator-sdk is an indirect dependency, we would want to ensure that `.Indirect != true` 

If operator-sdk is not a dep then this command will error with "go list -m: module github.com/operator-framework/operator-sdk: not a known dependency" though `wantevr` is handled if its null should we hide the error output and let the user know that operator-sdk is not a required dep?